UI/window-basic-main.cpp

```diff
@@ -84,6 +84,10 @@
 #include "windows.h"
 #endif
 
+#if defined(WHATSNEW_ENABLED)
```

Comment:
Generally we use `#ifdef` instead of `#if defined(...)` (unless it's unavoidable).
```suggestion
#ifdef WHATSNEW_ENABLED
```

Reply:
D'oh, yeah, also kind of obvious from the other lines... sleepy eyes.  : ]

Thanks!